docs/jwst/reset/description.rst

```diff
@@ -1,16 +1,16 @@
 Description
 ===========
+The reset correction is a MIRI step that attempts to corrects
+for the reset anomaly effect. This effect is caused by the non-ideal behavior of the FET upon resetting in the dark
+causing the initial frames in an integration to be offset from their expected values. Another MIRI effect caused by
+resetting the detectors is the RSCD effect (see the RSCD step documentation for more details). 
 
-.. note:: The reset step is not currently applied to MIRI exposures, but
-          will likely be reinstated in a future build.
 
 Assumptions
 -----------
-The reset correction is a MIRI-specific correction. It is
-assumed that the input science data have *NOT* had the zero group (or bias)
-subtracted. The reset correction should not remove the
-bias signal from the science exposure, therefore the reset correction
-for the first group is defined to be zero.
+The reset correction is a MIRI-specific correction it will not be applied to other instruments. 
```

Comment:
Need punctuation in the middle of this sentence, e.g. "... MIRI-specific correction. It will not ..." or "... MIRI-specific correction; it will not ..."